.github/workflows/promptflow-tracing-test.yml

```diff
@@ -0,0 +1,35 @@
+name: promptflow-tracing-test
+
+on:
+  schedule:
+    - cron: "40 18 * * *" # Every day starting at 2:40 BJT
+  pull_request:
+    paths:
+      - src/promptflow-tracing/**
+      - .github/workflows/promptflow-tracing-test.yml
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: check out repository
+        uses: actions/checkout@v4
+      - name: set up python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.9
+      - name: install poetry
+        uses: snok/install-poetry@v1
+        with:
+          virtualenvs-create: true
+          virtualenvs-in-project: true
+          installer-parallel: true
+      - name: install dependencies
+        run: poetry install --no-interaction
+        working-directory: ./src/promptflow-tracing
+      - name: run tests
+        run: |
+          poetry install --with test
+          poetry run pytest
```

Comment:
We have a run_coverage_test, could poetry replace that snippet to some extent?

Reply:
`run_coverage_test` actually passes args to pytest - can correct me if i'm wrong

so i'm wondering if we can deprecate that script and use native pytest to do so?

Reply:
updated the `pyproject.toml` and corresponding workflow, may take a look.